lib/rules/generator-star-spacing.js

```diff
@@ -55,6 +55,24 @@ module.exports = {
         var sourceCode = context.getSourceCode();
 
         /**
+         * Gets `*` token from a given node.
+         *
+         * @param {ASTNode} node - A node to get `*` token. This is one of
+         *      FunctionDeclaration, FunctionExpression, Property, and
+         *      MethodDefinition.
+         * @returns {Token} `*` token.
+         */
+        function getStarToken(node) {
+            var token = sourceCode.getFirstToken(node);
+
+            while (token.value !== "*") {
```

Comment:
I got an exception with this code: 
```js
TypeError: Cannot read property 'value' of undefined
```
Not really sure what I missed when upgrading `eslint` to the latest one since this PR was merged 13 days ago and no  one reported this.